server/autotest_server/testers/__init__.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-_TESTERS = ("custom", "haskell", "java", "jupyter", "py", "pyta", "racket")
+_TESTERS = ("custom", "haskell", "java", "jupyter", "py", "pyta", "racket", "r")
```

Comment:
Small thing, but in terms of alphabetical ordering I think `"r"` goes before `"racket"` (similar to py/pyta).